backend/languages/java.js

```diff
@@ -0,0 +1,35 @@
+const util = require("util");
+const exec = util.promisify(require("child_process").exec);
+var fs = require("fs");
+
+const java = (input, res, name) => {
+  fs.writeFile(`${name}.txt`, input, (err) => {
+    if (err) res.json({ error: err });
+    exec("docker run -d -it java:v1 /bin/bash").then((resp) => {
+      var id = resp.stdout.substring(0, 12);
+      exec(
+        `docker cp ${name}.java ${id}:/usr/src/app/test.java && docker cp ${name}.txt ${id}:/usr/src/app/input.txt && docker exec ${id} bash -c "javac test.java && java test<input.txt"`,
+        { timeout: 15000, maxBuffer: 50000 }
+      )
+        .then((resp) => {
+          res.json(resp);
+          exec(`rm ${name}.java && rm ${name}.txt`).then((resp) =>
+            console.log("Files removed")
+          );
+          exec(`docker kill ${id}`).then((resp) =>
+            console.log("Container Stopped")
+          );
+        })
+        .catch((err) => {
+          res.json(err);
+          exec(`rm ${name}.java && rm ${name}.txt`).then((resp) =>
+            console.log("Files removed")
+          );
+          exec(`docker kill ${id}`).then((resp) =>
+            console.log("Container Stopped")
```

Comment:
@rajatmaheshwari2512 you have written many console logs everywhere. Could you remove them?

Reply:
I feel like the console logs are important to understand promise resolution in case of an error in execution of the shell commands, since they can sometimes fail. If need be I can remove them and open a separate PR